### PR TITLE
DDCE-775:National Insurance number Question page

### PR DESCRIPTION
**Pull request by manpreetsingh1000**
# DDCE-(775)

##Bug fix / New feature (delete as appropriate)

Please include a summary / description of the change and which issue it fixes.  Include any relevant user needs, context or links to other PRs related to this PR (eg. acceptance tests, environment config).

## PR Suggestions
- Have you looked at the JIRA story to make sure all Acceptance Criteria has been met?
- Have you done a visual check of the changes?
- Is there any still commented or unused code? Lingering printlns?
- Have things been implemented in a complicated way?
- Are the test still over the coverage threshold?
- Does the compiler throw a lot of warnings? 
- Have methods and tests been named clearly?
- Were there any changes in the config? Do changes need to be made in app-config-???

## Checklist PR Raiser
 - [x]  I've made every effort to commit high quality, clean code and I have executed relevant static analyses to be sure
 - [x]  I've included appropriate tests with any code I've added (Unit, Integration, Acceptance etc.)
 - [x]  I've executed the acceptance test pack locally to ensure there are no functional regressions (Performance tests where relevant)
 - [x]  I've squashed my commits - including the JIRA issue number in the commit message
 - [ ]  I've run a dependency check to ensure all dependencies are up to date

## Checklist PR Reviewer
 - [ ]  I've made every effort to commit high quality, clean code and I have executed relevant static analyses to be sure
 - [ ]  I've included appropriate tests with any code I've added (Unit, Integration, Acceptance etc.)
 - [ ]  I've executed the acceptance test pack locally to ensure there are no functional regressions (Performance tests where relevant)
 - [ ]  I've squashed my commits - including the JIRA issue number in the commit message
 - [ ]  I've run a dependency check to ensure all dependencies are up to date

### CODE REVIEW

File: conf/messages.cy [hunk 1]
```diff
@@ -610,6 +610,12 @@ authorisedOfficialsName.lastName.error.required = (W)Enter the authorised offici
 authorisedOfficialsName.lastName.error.length = (W)The authorised official''s last name must be 100 characters or fewer(W)
 authorisedOfficialsName.lastName.error.format = (W)The authorised official''s last name must not include the following characters: & ( ) : @(W)
 
+# Authorised official position Page Messages
+# ----------------------------------------------------------
+isAuthorisedOfficialPosition.title = Does the authorised official have a national insurance number?
```
Comment (abhijeetardale) [inline conf/messages.cy:615]: welsh configuration is missing

File: app/navigation/AuthorisedOfficialsNavigator.scala [hunk 2]
```diff
@@ -39,6 +39,12 @@ class AuthorisedOfficialsNavigator @Inject()(implicit frontendAppConfig: Fronten
       case _ =>  routes.SessionExpiredController.onPageLoad()
     }
 
+    case IsAuthorisedOfficialPositionPage(index) => userAnswers: UserAnswers => userAnswers.get(IsAuthorisedOfficialPositionPage(index)) match {
+      case Some(true) => routes.IndexController.onPageLoad() // TODO redirect to next page once created
```
Comment (abhijeetardale) [inline app/navigation/AuthorisedOfficialsNavigator.scala:43]: can we combine this like some(_) with dead end controller? with to do comment

File: conf/messages.cy (hunk 1 above)
Comment (manpreetsingh1000) [reply conf/messages.cy]: added the configuration

File: app/navigation/AuthorisedOfficialsNavigator.scala (hunk 2 above)
Comment (manpreetsingh1000) [reply app/navigation/AuthorisedOfficialsNavigator.scala]: Updated as per comment

**Pull request merged**